README.md

```diff
@@ -16,11 +16,109 @@ var backlight = require('rpi-backlight');
 
 backlight.powerOn();
 backlight.powerOff();
+// 1 = Screen is OFF
+// 0 = Screen is ON
+backlight.getActualPowerValue();
+
+backlight.setMaxBrightness();
+backlight.getActualBrightness();
+// The screen goes Off at <= 9 brightness value
+backlight.setBrightnessValue(value);
+
+// Work in progress
+backlight.goToBrightnessValue(toValue, speed);
+
 ```
 
-## Coming soon
+## Forked from
+
+- [pimterry/rpi-backlight](https://github.com/pimterry/rpi-backlight)
+
+## Example of a simple app
```

Comment:
This example is good, and it's a great idea, but I don't think the source needs to live in the readme. Can you put these files in an `example/` folder, and then just link to them?

Reply:
If I understand correctly, I create a folder directly inside rpi-backlight folder?